internal/grid/connection.go

```diff
@@ -175,7 +175,7 @@ func (c ContextDialer) DialContext(ctx context.Context, network, address string)
 }
 
 const (
-	defaultOutQueue    = 65535    // kind of close to max open fds per user
+	defaultOutQueue    = 1048576  // (ulimit -Hn) - kind of close to max open fds per user
```

Comment:
```suggestion
	defaultOutQueue    = 65535
```
This has nothing to do with FDs, and is a *per connection* property, so there is no need to scale it.

You are just wasting memory.